packages/@webex/plugin-meetings/src/meeting/muteState.ts

```diff
@@ -9,7 +9,8 @@ import {AUDIO, VIDEO} from '../constants';
    If we ever need to support it, search for REMOTE_MUTE_VIDEO_MISSING_IMPLEMENTATION string to find the places that need updating
 */
 
-const createMuteState = (type, meeting, mediaDirection) => {
+// eslint-disable-next-line import/prefer-default-export
+export const createMuteState = (type, meeting, mediaDirection) => {
```

Comment:
I had to change this so that I could mock this function more easily in unit tests